### ISSUE TITLE
phpdoc_params does not works correctly

### ISSUE THREAD

**Issue by ptcong** [FriendsOfPHP/PHP-CS-Fixer#1252]
I use the fixers: `phpdoc_params` and `phpdoc_scalar`
First time my phpdoc is 
```php
/**
 * @param integer $limit
 * @param string  $more
 *
 * @return array
 */
```
After fixer run, it become
```php
/**
 * @param int $limit
 * @param string  $more
 *
 * @return array
 */
```
then i tried to change it to check whether phpdoc_params works ?
```php
/**
 * @param int       $limit
 * @param string  $more
 *
 * @return array
 */
```
But after fixer run, nothing happened. params not be aligned as expected.

**Comment by SpacePossum** [comment]
Hi,
I failed to reproduce your case on 1.9 and master. Which version of the Fixer are you using and what version PHP/HHVM?
I tried:
`php php-cs-fixer fix test.php -vvv --fixers=phpdoc_params,phpdoc_scalar` and at Symfony level against your first code block.

**Comment by ptcong** [comment]
I'm using fixer 1.8.1, php 5.4 and running with  psr2 level

**Comment by GrahamCampbell** [comment]
I can't reproduce this.

**Comment by ptcong** [comment]
Ah, i just found that the issues not caused if i run fixer for a class file.
The issues caused when i use for `functions.php` file.

**Comment by keradus** [comment]
Could you provide a fully working code example that can be used to
duplicate a bug ?

**Comment by SpacePossum** [comment]
```php
<?php
/**
 * @param   int       $limit
 * @param   string  	 $more
 *
 * @return array
 */
function test($limit, $more)
{
    return array($limit, $more);
}
```

`
php php-cs-fixer fix test.php -vvv --fixers=phpdoc_params,phpdoc_scalar
`

Confirmed on 1.9 that this doesn't get fixed.

**Comment by GrahamCampbell** [comment]
What happens if you put the fixers the other way around out of interest?

**Comment by SpacePossum** [comment]
The `scaler` fixer keeps working, the `params` doesn't.

**Comment by ptcong** [comment]
`scalar` still works, i have tried with each fixer, but params does not.

**Comment by GrahamCampbell** [comment]
Right, but it works if you run php-cs-fixer twice on the code base, so we've just got the fixer priority wrong internally?

**Comment by ptcong** [comment]
I tried many times, but `params` does not works, even with @SpacePossum code or my code.

**Comment by GrahamCampbell** [comment]
Hmm, that's very strange. :(

**Comment by SpacePossum** [comment]
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1255
PR to ensure the order (not a fix BTW)

**Comment by GrahamCampbell** [comment]
I might rewrite the params fixer next month as the current implementation is pretty complicated and hard to follow.

**Comment by keradus** [comment]
@ptcong please take a look at #1287 ;)

**Comment by ptcong** [comment]
Ya! it works now.
Thank you @keradus

**Comment by keradus** [comment]
Closing due to merge of #1287
